docs/README.md

```diff
@@ -1,12 +1,16 @@
 # Rogue [![StyleCI](https://styleci.io/repos/64166359/shield?style=flat-rounded)](https://styleci.io/repos/64166359)
 
-This is **Rogue**, the DoSomething.org user activity service. Rogue is built using [Laravel 5.5](https://laravel.com/docs/5.5) and [React](http://reactjs.com).
+This is **Rogue**, the DoSomething.org user activity service. Rogue is built using [Laravel 6](https://laravel.com/docs/6.x) and [React](http://reactjs.com).
+
+-   [API Documentation](./endpoints)
 
 ### Getting Started
 
-To get started with development, follow the [installation](./development/installation.md) and [contributing](./development/contributing.md) documentation.
+To get started with development, check out our communal docs for installing [Homestead](https://github.com/DoSomething/communal-docs/tree/master/Homestead).
+
+After installation, run `php artisan db:seed` to create sample campaign and activity data.
```

Comment:
This is out of scope, but this got me thinking it could be nice to embellish this with something akin to our [Northstar setup](https://github.com/DoSomething/northstar#contributing)?

Reply:
Yeah, i agree. I can clean this up a bit in a follow up PR that 🔥 a lot of the outdated (or duplicated by communal-docs/docs)